personal/user-overrides.js

```diff
@@ -55,8 +43,12 @@ user_pref("browser.search.update", false); // do not update opensearch engines
 user_pref("network.dns.skipTRR-when-parental-control-enabled", false); // bypass parental controls when using DoH
 user_pref("network.trr.confirmationNS", "skip"); // skip TRR confirmation request
 user_pref("extensions.webextensions.restrictedDomains", ""); // remove Mozilla domains so adblocker works on pages
+user_pref("identity.fxaccounts.enabled", false); // disable Firefox Sync
+user_pref("browser.tabs.firefox-view", false); // disable Firefox View

```

Comment:
Not sure if this is relevant, but [this pref no longer does anything](https://bugzilla.mozilla.org/show_bug.cgi?id=1787443).

Reply:
was missing `browser.tabs.firefox-view-next`